src/components/subnavigation/developer-topics.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import {
+    DeveloperTopicsTab,
+    StyledButton,
+} from '../../styled/developer-topics';
+import { Context } from './navbar';
+
+const DeveloperTopics: React.FunctionComponent = () => {
+    const { open, setIsOpen } = useContext(Context);
+    const handleOpening = () => {
+        setIsOpen(!open);
+    };
+    return (
+        <DeveloperTopicsTab>
+            <StyledButton onClick={handleOpening}>
+                Developer Topics
```

Comment:
So I noticed on Figma that the developer topics menu also closes when we click outside of the menu. I'm not sure if that's the desired behavior in the real design, but just figured I'd point it out in case it comes up.